### DISCUSSION TITLE
Grafana query for chaos mesh gives error.api.bad_request: token is empty

### DISCUSSION THREAD

**Discussion by sangugwad1993** [chaos-mesh/chaos-mesh#4105] [Q&A]
[](url)Performed below steps:
1. Have installed Docker, and Kubernetes and deployed Chaos-mesh pods using helm 
2. On the physical system added chaosd services
3. On Grafana added Chaos-mesh plugin
4. Followed the procedure from the chaos mesh document and still getting the below error on grafan:

<img width="736" alt="bad api error" src="https://github.com/chaos-mesh/chaos-mesh/assets/137896048/e21e577d-eb1b-4b03-b143-f566ae5e2f28">

**Comment by fzyzcjy** [comment]
Hi, did you solve the problem? Thanks

**Reply by sangugwad1993** [reply]
No, we haven't. Need help here. Can we get a solution for this?

**Reply by fzyzcjy** [reply]
I am not a maintainer of ChaosMesh... :/
